lib/vdc/DerivedVar.cpp

```diff
@@ -361,7 +361,7 @@ void resampleToStaggered(
 		inDims.push_back(inMax[i]-inMin[i]+1);
 		outDims.push_back(outMax[i]-outMin[i]+1);
 	}
-	size_t sz = std::max(vproduct(outDims), vproduct(inDims));
+	size_t sz = (std::max)(vproduct(outDims), vproduct(inDims));
```

Comment:
This syntax looks weird. What problem does it fix?

Reply:
The same comment applies to the rest changes in this file.

Reply:
Windows.h has a macro that overrides the max token, std::max will evaluate into the expansion of the macro.  Putting it in parenthesis prevents the compiler from inserting the macro.

Reply:
You could undefine the macro